stories/chart.stories.tsx

```diff
@@ -80,15 +92,26 @@ storiesOf('HURUmap UI|Charts/BarChart', module)
         <BarChart
           width={number('width', 500)}
           height={number('height', 300)}
-          labels={datum => `${datum.y}${text('dataUnit', '%')}`}
+          labels={datum =>
+            `Group ${datum.tick} Tick\n${datum.x} ${datum.y}${text(
+              'dataUnit',
+              '%'
+            )}`
+          }
```

Comment:
So no more labels support ontop of bars ?

Reply:
Answered above.